src/ruleLoader.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function () {
+  const callback = this.async();
+  const { plugin } = this.query;
+
+  plugin.tags.promise
+    .then(tags => {
+      callback(null, "module.exports = ['" + tags.join( "', '" ) + "'];");
```

Comment:
Could you use string templates instead?

`module.exports = ['${tags.join("', '")}'];`

Reply:
Done